internal/kubernetes/manifest.go

```
@@ -0,0 +1,155 @@
+	}
+	if strings.TrimSpace(opts.Image) == "" {
+		return nil, errors.New("installer image is required")
+	}
+	if strings.TrimSpace(opts.CDIKind) == "" {
+		opts.CDIKind = DefaultCDIKind
+	}
+	if opts.DeviceCount <= 0 || opts.DeviceCount > 8 {
+		return nil, fmt.Errorf("device count must be between 1 and 8, got %d", opts.DeviceCount)
+	}
+	if len(strings.TrimSpace(string(opts.ConfigYAML))) == 0 {
+		return nil, errors.New("Mock NVML config is required")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)